utilities/vcf2json.py

```diff
@@ -281,27 +285,32 @@ def getMultADs(output_json, record, sample_position, alt, alt_ad_index, hasAD, n
 # Processes SnpEff and SnpSift output into final json file.
 
 
-def extractINFOField(output_json, record, codeDict):
+def extractINFOField(variant_id, record, codeDict):
     # Info field is dict with entries: ANN (SnpEff output) and POPAF (gnomAD output)
-
     if 'ANN' in record.INFO:
-        output_json["molecConseq"] = []
+        # output_json["molecConseq"] = []
+        molecular_json = []
```

Comment:
I believe there can be a POPAF stat in the INFO field without an ANN field. Should this declaration be moved to outside the first if statement?

Reply:
Corrected it.